apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/EditProfileForm.tsx

```diff
@@ -102,7 +102,7 @@ const EditProfileForm = ({ profile }: { profile: ProfileModel }) => {
       avatarUrl: session?.user?.image || null,
       linkedIn: values.linkedin,
       bio: values.bio,
-      cvUrl: values.cvUrl,
+      cvUrl: values.cvUrl ?? profile.cvUrl,
```

Comment:
**nit:** I think it should not be neccesary here, as values.cvUrl should be already set when initalizing from data